### ISSUE TITLE
Split padding fields to vertical/horizontal 

### ISSUE THREAD

**Issue by chrisdola** [siddharthroy12/recoded#15]
It would be incredibly helpful to split out the padding fields to horizontal & vertical to allow for more custom sizing by default. As it stands now, the only way to handle this after the fact would be editing the mp4 to crop the size once it's done. 

I already made the changes locally and can submit a PR.. Just wanted to get input before submitting it.  😊

**Comment by siddharthroy12** [comment]
Yeah. You can go ahead and submit a PR. Contributions are always welcomed
